src/amber/cli/templates/app/src/assets/javascripts/main.js

```diff
@@ -1 +1,23 @@
 import Amber from 'amber'
+
+if (!Date.prototype.toGranite) {
```

Comment:
I spend very little time javascripting so perhaps my instincts are off, but this method name feels weird to me. It's not emitting a Granite, it's emitting a formatted string representation of a datetime.

Reply:
@robacarp I agree.  This should be a more generic name.  `toIso8601()` or `toDbFormat()`?

Reply:
I didn't check the code for the other ORMs for Amber, I am using granite ORM for my project, but if they all expect the same time-date format then I agree toDbFormat() or toAmberFormat() would be a better name. I think toISO8601() is slightly confusing because 8601 format is 2018-02-15T12:00:00 with the 'T' separator and also may be confused with toISOString().  I did originally call it toGraniteString() but thought the 'String' was redundant because if you use this patched in method you will be aware of what you are creating. I can rename it to whatever you think is the best name.

Reply:
Perhaps Granite should learn to parse datetimes from one of the iso8601 formats.

Reply:
yes that would be a good option